### DISCUSSION TITLE
@types/react-dom

### DISCUSSION THREAD

**Discussion by samuelcastro** [UnlyEd/next-right-now#267] [General] [answered]
Why did you remove `@types/react-dom`?

**Comment by Vadorequest** [comment, marked as answer]
I don't think I removed them, I believe they were never added in the first place.

**Reply by samuelcastro** [reply]
Oh got it.

Why not add it?

**Reply by Vadorequest** [reply]
I don't know, probably was forgotten! I never had a need for it I guess, never noticed it was missing either. I'll add it in the next big PR.
